recipes/pytorchrl/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pytorchrl" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pytorch/rl/archive/v{{ version }}.tar.gz
+  sha256: 8851a84316f2a1d61d23ec753c90b545bc6479890ec65c5278987ae7c8a2ceec
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win]
+  missing_dso_whitelist:
+    - $RPATH/libtorch_python{{ SHLIB_EXT }}  # [not win]
```

Comment:
```suggestion
  rpaths:
    - lib
    - {{ SP_DIR }}/torch
```

Reply:
@isuruf with this modification I again get the following error:
```
Warning: rpath /home/conda/staged-recipes/build_artifacts/pytorchrl_1719418355124/_build_env/lib is outside prefix /home/conda/staged-recipes/build_artifacts/pytorchrl_1719418355124/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p (removing it)
   INFO: sysroot: '/home/conda/staged-recipes/build_artifacts/pytorchrl_1719418355124/_build_env/x86_64-conda-linux-gnu/sysroot/' files: '['usr/share/zoneinfo/zone1970.tab', 'usr/share/zoneinfo/zone.tab', 'usr/share/zoneinfo/tzdata.zi', 'usr/share/zoneinfo/right/Zulu']'
   INFO (pytorchrl,lib/python3.10/site-packages/torchrl/_torchrl.so): Needed DSO lib/libc10.so found in conda-forge/linux-64::libtorch==2.3.1=cpu_mkl_h0bb0d08_100
   INFO (pytorchrl,lib/python3.10/site-packages/torchrl/_torchrl.so): Needed DSO lib/libtorch_cpu.so found in conda-forge/linux-64::libtorch==2.3.1=cpu_mkl_h0bb0d08_100
  ERROR (pytorchrl,lib/python3.10/site-packages/torchrl/_torchrl.so): $RPATH/libtorch_python.so not found in packages, sysroot(s) nor the missing_dso_whitelist.
.. is this binary repackaging?
   INFO (pytorchrl,lib/python3.10/site-packages/torchrl/_torchrl.so): Needed DSO lib/libstdc++.so.6 found in conda-forge/linux-64::libstdcxx-ng==13.2.0=hc0a3c3a_13
   INFO (pytorchrl,lib/python3.10/site-packages/torchrl/_torchrl.so): Needed DSO lib/libgcc_s.so.1 found in conda-forge/linux-64::libgcc-ng==13.2.0=h77fa898_13
   INFO (pytorchrl,lib/python3.10/site-packages/torchrl/_torchrl.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_14
   INFO (pytorchrl,lib/python3.10/site-packages/torchrl/_torchrl.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/ld-linux-x86-64.so.2 found in CDT/compiler package conda-forge/noarch::sysroot_linux-64==2.17=h4a8ded7_14
WARNING (pytorchrl): run-exports library package conda-forge/linux-64::pytorch==2.3.1=cpu_mkl_py310h75865b9_100 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (pytorchrl): interpreter (Python) package conda-forge/linux-64::python==3.10.14=hd12c33a_0_cpython in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (pytorchrl): run-exports library package conda-forge/linux-64::numpy==2.0.0=py310h515e003_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```